test/unit/specs/pages/collectionspage.spec.js

```diff
@@ -1,7 +1,10 @@
 import CollectionsPage from '@/pages/CollectionsPage';
```

Comment:
🎉 Thank you for handling testing!

Reply:
Seconded, great job keeping this in mind!